Dockerfile.tester

```diff
@@ -5,10 +5,11 @@ MAINTAINER Daniël van Gils
 #get all the go testing stuff
 RUN go get github.com/onsi/ginkgo/ginkgo
 RUN go get github.com/onsi/gomega
+RUN go get github.com/sirupsen/logrus
```

Comment:
Aha! I was wondering why go couldn't find the vendored Sirupsen/logrus... It turns out the author renamed his github username from capitalized to not.

https://github.com/sirupsen/logrus/issues/570

So, good catch. Probably we need both `sirupsen/logrus` either vendored or installed sysem-wide to make it work. Make sense.